packages/form-js-editor/src/render/components/properties-panel/Util.js

```diff
@@ -1,59 +1,59 @@
-import { debounce as _debounce } from 'min-dash';
-
-export function arrayAdd(array, index, item) {
-  const copy = [ ...array ];
-
-  copy.splice(index, 0, item);
-
-  return copy;
-}
-
-export function arrayRemove(array, index) {
-  const copy = [ ...array ];
-
-  copy.splice(index, 1);
-
-  return copy;
-}
-
-export function debounce(fn) {
-  if (isTest()) {
-    return fn;
-  }
-
-  return _debounce(fn, 300);
-}
-
-export function prefixId(id) {
-  return `fjs-properties-panel-${ id }`;
-}
-
-export function stopPropagation(listener) {
-  return (event) => {
-    event.stopPropagation();
-
-    listener(event);
-  };
-}
-
-export function textToLabel(text) {
-  if (text.length > 10) {
-    return `${ text.substring(0, 10) }...`;
-  }
-
-  return text;
-}
-
-export const INPUTS = [
-  'checkbox',
-  'number',
-  'radio',
-  'select',
-  'textfield'
-];
-
-function isTest() {
-
-  // @ts-ignore-next-line
-  return window.__env__ && window.__env__.NODE_ENV === 'test';
+import { debounce as _debounce } from 'min-dash';
+
+export function arrayAdd(array, index, item) {
+  const copy = [ ...array ];
+
+  copy.splice(index, 0, item);
+
+  return copy;
+}
+
+export function arrayRemove(array, index) {
+  const copy = [ ...array ];
+
+  copy.splice(index, 1);
+
+  return copy;
+}
+
+export function debounce(fn) {
+  if (isTest()) {
```

Comment:
Just spotted this one. 

Where do we us it? I believe it is an anti-pattern to hide such switch in a utility. Alternative: Configure debouce (or not) where it is being used, cf. https://github.com/bpmn-io/diagram-js/blob/develop/lib/core/Canvas.js#L154 for example.

Reply:
Done (cf. https://github.com/bpmn-io/form-js/pull/75/commits/0774d057583cd07c815b7591a11bc3b07d5987d9).